### ISSUE TITLE
Dropout Layer in OpenAIGPTMultipleChoiceHead not used

### ISSUE THREAD

**Issue by pglock** [huggingface/pytorch-pretrained-BERT#354]
[OpenAIGPTMultipleChoiceHead](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/modeling_openai.py#L363) defines an additional dropout layer, which is not used in `forward`.
